Library/ViewModels/ShareViewModel.swift

```diff
@@ -1,10 +1,8 @@
-#if os(iOS)
 import KsApi
 import LiveStream
 import Prelude
 import ReactiveSwift
 import Result
-import Social
```

Comment:
We use `UIActivityViewController` for all of this now, no need to import `Social`.